level1/1-4.java

```diff
@@ -0,0 +1,94 @@
+import java.util.*;
+import java.util.stream.Collectors;
+
+class Solution {
+    Map<String, Integer> leftPointer = new HashMap<String, Integer>();
+    Map<String, Integer> rightPointer = new HashMap<String, Integer>();
+    List<String> resultHandList = new ArrayList<String>();
+
+    public String solution(int[] numArr, String hand) {
+
+        List<Integer> numList = Arrays.asList(Arrays.stream(numArr).boxed().toArray(Integer[]::new));
+        
+        List<Integer> leftList = Arrays.asList(new Integer[]{1,4,7});
+        List<Integer> rightList = Arrays.asList(new Integer[]{3,6,9});
+
+        // leftStatus init
+        leftPointer.put("x", 3);
+        leftPointer.put("y", 0);
+
+        // rightStatus init
+        rightPointer.put("x", 3);
+        rightPointer.put("y", 2);
+
+        int[][] arr2d = {
+                {1, 2, 3},
+                {4, 5, 6},
+                {7, 8, 9},
+                {10, 0, 12}
+        };
+
+        numList.stream().forEach(
+                e -> {
+                    // 왼손/오른손 배열인지 한번 확인 후 for문 돌림
+
+                    int leftRange = leftList.indexOf(e);
+                    int rightRange = rightList.indexOf(e);
```

Comment:
이건 컨벤션을 생각해보는게 좋을것 같아요!
```suggestion
        numList.stream().forEach( e -> {
            // 왼손/오른손 배열인지 한번 확인 후 for문 돌림
            int leftRange = leftList.indexOf(e);
            int rightRange = rightList.indexOf(e);
```